On aarch64, this tests takes ~0.4s, which is larger than the default
threshold. Increase it a bit to let this test pass.
diff --git a/test/test_speed.py b/test/test_speed.py
index f3fa7c7b3d..ba5784acc8 100644
--- a/test/test_speed.py
+++ b/test/test_speed.py
@@ -30,7 +30,7 @@ def _check_speed(time_per_run, number=4, run_warm=True):
     return decorated
 
 
-@_check_speed(0.3)
+@_check_speed(0.5)
 def test_os_path_join(Script):
     s = "from posixpath import join; join('', '')."
     assert len(Script(s).completions()) > 10  # is a str completion
